src/Illuminate/Database/Connection.php

```diff
@@ -1545,6 +1545,18 @@ public function unsetTransactionManager()
         $this->transactionsManager = null;
     }
 
+    /**
+     * Ignores the current transaction level when calling the callbacks.
+     *
+     * @return void
+     */
+    public function ignoreLatestTransactionForCallbacks()
+    {
+        $this->transactionsManager?->callbacksShouldIgnore(
+            $this->transactionsManager?->getTransactions()?->last()
```

Comment:
The switch from pulling the `->first()` transaction to `->last()` was intentional. Otherwise, it keeps re-adding the first transition to the list. We want to add the most recent, so we're taking the last instead.

Reply:
In Octane environment, should `$this->transactionsManager` get unset via `disconnect()` method? 

https://github.com/laravel/octane/blob/2.x/src/Listeners/DisconnectFromDatabases.php#L15C26-L15C36

Reply:
Yeah, I've changed the implementation. We won't have to clean it up (it's gone now)